projects/ngrx.io/content/guide/data/entity-actions.md

```diff
@@ -0,0 +1,269 @@
+
+## Tagging the EntityAction
+
+A well-formed action `type` can tell the reader what changed and
+who changed it.
+
+The NgRx Data library doesn't look at the type of an `EntityAction`,
+only its `entityName` and `entityOp`.
+So you can get the same behavior from several different actions,
+each with its own informative `type`, as long as they share the same
+`entityName` and `entityOp`.
+
+The optional `tag` parameter of the `EntityActionFactory.create()` method makes
+it easy to produce meaningful _EntityActions_.
+
+You don't have to specify a tag. The `entityName` is the default tag that appears between brackets in the formatted `type`,
+e.g., `'[Hero] NgRx Data/query-all'`.
+
+Here's an example that uses the injectable `EntityActionFactory` to construct the default "query all heroes" action.
+
+```typescript
+const action = entityActionFactory<Hero>(
+  'Hero', EntityOp.QUERY_ALL, null, 'Load Heroes On Start'
+);
+
+store.dispatch(action);
+```
+
+Thanks to the _NgRx Data _Effects_, this produces _two_ actions in the log, the first to initiate the request and the second with the successful response:
+
+```sh
+[Hero] NgRx Data/query-all
+[Hero] NgRx Data/query-all-success
+```
+
+This default `entityName` tag identifies the action's target entity collection.
+But you can't understand the _context_ of the action from these log entries. You don't know who dispatched the action or why.
+The action `type` is too generic.
+
+You can create a more informative action by providing a tag that
+better describes what is happening and also make it easier to find
+where that action is dispatched by your code.
+
+For example,
+
+```typescript
+const action = entityActionFactory<Hero>(
+  'Hero', EntityOp.QUERY_ALL, null, 'Load Heroes On Start'
+);
+
+store.dispatch(action);
+```
+
+The action log now looks like this:
+
+```sh
+[Load Heroes On Start] NgRx Data/query-all
+[Load Heroes On Start] NgRx Data/query-all-success
+```
+
+### Handcrafted EntityAction
+
+You don't have to create entity actions with the `EntityActionFactory`.
+Any action object with an `entityName` and `op` property is
+an entity action, as explained [below](#where-are-the-entityactions).
+
+The following example creates the initiating "query all heroes" action by hand.
+
+```typescript
+const action = {
+  type: 'some/arbitrary/action/type',
+  entityName: 'Hero',
+  op: EntityOp.QUERY_ALL
+};
+
+store.dispatch(action);
+```
+
+It triggers the HTTP request via _NgRx Data effects_, as in the previous examples.
+
+Just be aware that _NgRx Data effects_ uses the `EntityActionFactory` to create the second, success Action.
+Without the `tag` property, it produces a generic success action.
+
+The log of the two action types will look like this:
+
+```sh
+some/arbitrary/action/type
+[Hero] NgRx Data/query-all-success
+```
+
+<a id="where-are-the-entityactions"></a>
```

Comment:
Should we leave these anchor tags out? 
Because every title gets an anchor.

Reply:
Good catch. I removed the anchors that merely duplicated the titles.